src/odemis/driver/static.py

```diff
@@ -109,6 +117,31 @@ def __init__(self, name, role, mag, mag_choices=None, na=0.95, ri=1,
                                                       range=((0, 0), (1e6, 1e6)),
                                                       cls=numbers.Real,
                                                       unit="px")
+        # mirrorPositionTop and mirrorPositionBottom are similar. They represent
+        # a line on the CCD as a function of the wavelength (increasing along the X axis).
+        # So it's two values, a & b, which define a line as px = a + b * wl.
+        # Where px has origin at the top of the CCD. The typical image has a field
+        # in the order ~100 nm. So the a & b values can get quite large, especially
+        # when the user is playing with drawing the lines. Hence, we put very large
+        # range. The main goal of using a TupleContinuous over a TupleVA is to
+        # check it's 2 floats.
+        if mirror_pos_top is not None:
+            if (not isinstance(pole_pos, collections.Iterable) or
+                len(pole_pos) != 2 or any(not isinstance(v, numbers.Real) for v in mirror_pos_top)):
+                raise ValueError("pole_pos must be 2 floats, got %s" % mirror_pos_top)
+            self.mirrorPositionTop = model.TupleContinuous(tuple(mirror_pos_top),
+                                                      range=((-1e18, -1e18), (1e18, 1e18)),
+                                                      cls=numbers.Real,
+                                                      unit="px, px/m")
```

Comment:
does this mean the unit of the first entry is px and of the second is px/m? If yes, can you add this to the docstring?